### DISCUSSION TITLE
TypeORM migration to Prisma

### DISCUSSION THREAD

**Discussion by edilson** [prisma/prisma#6364] [Q&A] [answered]
Hello, currently I'm using TypeORM but I intend to migrate to Prisma, I wanted to do this gradually and for that I need to, for some time, use both at the same time. Is it possible to do it? How could I achieve that?

**Comment by nikolasburk** [comment, marked as answer]
Hey @edilson 👋 

> I  wanted to do this gradually and for that I need to, for some time, use both at the same time.

Yes this is absolutely possible and we've seen a lot of people successfully migrate from TypeORM to Prisma and gradually introducing Prisma in their projects.

We have a migration guide in our documentation that describes this process: [Migrate from TypeORM](https://www.prisma.io/docs/guides/migrate-to-prisma/migrate-from-typeorm)

This article could also be useful for you to learn directly from someone who went through such a migration in the past: [Migrating a Large Production App From TypeORM To Prisma](https://darioielardi.dev/migrating-a-large-production-app-from-typeorm-to-prisma)

Feel free to raise any further questions here as a GitHub Discussion and/or on the [Prisma Slack](https://slack.prisma.io) 🙂

**Reply by edilson** [reply]
Great @nikolasburk, thanks a lot for your support. This article helped me a lot.

**Reply by georgekrax** [reply]
But how can I create **utility functions** in order to not pass every time the model as a function parameter?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
